### PR TITLE
Flesh out scenarios and add feedback from csswg

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,26 +11,30 @@
...
+We hope to accomplish this by introducing a small addition to CSS.
 
 We propose to introduce a new value for `transform-style`: `detached`.
 
-This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`. The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
+This new property builds upon the existing 3D transforms that are shipping in all browsers and extends the behavior of `transform-style: preserve-3d`.
+The main difference is that instead of flattening the transformed elements back to the page, the transformed element stays in 3D space.
 If `transform-style: detached` is specified on an element, children with 3D transforms will be lifted out of the page.
 
 Here is an example rendering of the effect:
```
Comment (raviramachandra) [inline README.md]: Please give credits to original owner. (Josh Carpenter)

File: README.md [hunk 2]
```diff
@@ -11,26 +11,30 @@
...
 ![scene](https://github.com/rcabanier/detached_explainer/raw/master/detached.gif "Scene")
 
-As the page or a parent element scrolls, the detached element should scroll. CSS animations and transitions will apply as well.
+As the page or a parent element scrolls, the detached element should scroll.
+CSS animations and transitions will apply as well.
 
 Because we don't want to render content anywhere in the user's space, we also propose to define a "safe space" around the browser.
```
Comment (raviramachandra) [inline README.md]: Prefer: In order to disallow unrestricted access of user's space to the content developer, we also propose to define a "safe space" that shall be requested to the user by content developer and user consented.

File: README.md [hunk 3]
```diff
@@ -11,26 +11,30 @@
...
 Content can not be placed outside this space.
```
Comment (raviramachandra) [inline README.md]: Example seems to be out of place. I could not get a flow.  May be should come along with the proposals.

File: README.md [hunk 4]
```diff
@@ -64,44 +68,46 @@ Example:
...
-[//]: # (TODO: We need to explan this, but I cant think of the best explanation)
+Transform style `flat` and effects that cause flattening ([spec](https://drafts.csswg.org/css-transforms-2/#grouping-property-values)) will cause `detached` fall back to `preserve-3d` behavior in descendent elements.
 
+The one exception is root node which allows detached descendants even though an implicit `flat` transform-style is applied.
 
-### 3. Effects such as clip applied to parents of detached nodes are not applied to detached nodes.
+### 3. Only the top-most frame can use this feature.
 
-The detached elements and their descendants are rendered onto surfaces that are distinct from their parents. Any effect applied to parents will be applied during the compositing stage and will not be applied to the detached elements.
+iframes should not be allowed to created detached content.
```
Comment (raviramachandra) [inline README.md:87]: nit: we can do it: <iframe>, easier to read.

File: README.md [hunk 5]
```diff
@@ -64,44 +68,46 @@ Example:
...
+We propose that descendents with `detached` are treated as `preserve-3d`.
 
-[//]: # ([Tricky design choice 2])
-[//]: # ([etc.])
-
-## Considered alternatives
+[//]: # (TODO: We need to explain this, but I cant think of the best explanation)
 
+#
+## Considered alternatives / other details
```
Comment (raviramachandra) [inline README.md]: nit: Other Details can go into "Miscellaneous" or Considered Alternative & Other Details

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: will include in the commented out section 'references and acknowledgements' in the end

File: README.md (hunk 4 above)
Comment (ghost) [reply README.md:87]: I think we can leave it as-is. we don't decorate tags anywhere else in the explainer.

File: README.md [hunk 6]
```diff
@@ -64,44 +68,46 @@ Example:
...
 [//]: # ([This should include as many alternatives as you can, from high level architectural decisions down to alternative naming choices.])
 
+### 1. Applying the detached style directly to the surface to be detached.
```
Comment (raviramachandra) [inline README.md:108]: Really can't figure out the difference between Alternative 1 and 2. Need a more cleaner flow of explainer.

**Review by raviramachandra — COMMENTED** [review]
In general, the flow is confusing ?  It is not clear what we are proposing here.

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md:108]: the difference is not between alternative 1 and 2. The difference is between main proposal vs. alternative 1 and main proposal vs. alternative 2

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md]: updated.

File: README.md (hunk 3 above)
Comment (ghost) [reply README.md]: done.

File: README.md (hunk 5 above)
Comment (ghost) [reply README.md]: title changed

File: README.md (hunk 6 above)
Comment (ghost) [reply README.md:108]: title changed

**Comment by ghost** [comment]
rejecting this pr in favor of switching to transform attribute to specify detached

**Pull request closed without merging**